nltk/tbl/erroranalysis.py

```diff
@@ -32,9 +32,7 @@ def error_list(train_sents, test_sents):
             if train_pos != test_pos:
                 left = " ".join("%s/%s" % w for w in train_sent[:wordnum])
                 right = " ".join("%s/%s" % w for w in train_sent[wordnum + 1 :])
-                mid = "%s/%s->%s" % (word, test_pos, train_pos)
-                errors.append(
-                    "%25s | %s | %s" % (left[-25:], mid.center(22), right[:25])
-                )
+                mid = f"{word}/{test_pos}->{train_pos}"
```

Comment:
just remarking that `"%s/%s->%s" % (word, test_pos, train_pos)` to `f"{word}/{test_pos}->{train_pos}"` is such an improvement!